root/views/layout.sml

```diff
@@ -7,17 +7,17 @@ html
       meta(http-equiv='X-UA-Compatible' content='IE=edge, chrome=1')
       meta(name='description' content='')
       meta(name='author' content='<%= github_username %>')
-      //- uncomment if your site is responsive!
-      //- meta(name="viewport" content="width=device-width, initial-scale=1")
+      meta(name="viewport" content="width=device-width, initial-scale=1")
 
     block(name='title')
       title Title of Your Site
 
     block(name='stylesheets')
       link(rel='stylesheet' href='css/index.css')
 
-  body
-    block(name='content')
+  body(role='document')
```

Comment:
So I'm not sure we can authoritatively make the call here between `document` and [application](https://www.w3.org/TR/wai-aria/roles#application). Although I am no expert at aria roles at all. Can you explain a little further what went into your decision here?